api/schema/todo.py

```diff
@@ -0,0 +1,29 @@
+from pydantic import BaseModel, Field
+from typing import Optional
+from tortoise.contrib.pydantic import pydantic_model_creator
+from api.model.todo import Todo
+
+#Creating model
+GetTodo = pydantic_model_creator(Todo, name="Todo")
+
+
+"""
```

Comment:
In Python, class *docstrings* go inside the class body. If this is meant to be a comment, use `#`. See PEP 257 for more conventions.